tests/cross-compile/cpp-resource-reflection.slang

```diff
@@ -1,4 +1,4 @@
-//TEST:CPU_REFLECTION: -profile cs_5_0 -entry computeMain -target cpp 
+//TEST:CPU_REFLECTION: -profile cs_5_0 -entry computeMain -target cpp -no-codegen
```

Comment:
I may have missed it in the PR description, but I wonder why some of these compiles need `-no-codegen` now where they were okay without it before.

Reply:
Previously, `-no-codegen` was added for all tests implicitly in the reflection test program. Now it's not, as I need to do code generation for the used bindings test. I guess I could drop this option on all/most of the other tests too, and update the expected outputs, which would result in better coverage.